src/coreclr/jit/importer.cpp

```diff
@@ -9455,6 +9455,44 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
 
         switch (callInfo->kind)
         {
+            case CORINFO_VIRTUALSTATICCALL:
```

Comment:
This looks almost like `CORINFO_CALL_CODE_POINTER` - can't we use that one?

Reply:
Thanks Michal for the suggestion. I originally tried that and hit some issues but now I tried to just switch it over and it works fine so I assume the problems I was hitting were unrelated and I changed it to <code>CORINFO_CALL_CODE_POINTER</code> per your suggestion in the latest commit.